parser/test.m

```diff
@@ -0,0 +1,40 @@
+function [output1, output2] = test(arg1, arg2)
```

Comment:
isn't the convention to call test functions with suffix as the function to test, i.e.: `test_parse_doc.m`?

Reply:
didn't know that. I'll change it

Reply:
i don't know either-- i'm basing it just from python's pytest-- obviously things might be different in matlab-- please inform yourself how those tests are run-- @po09i @gab-berest @rtopfer might know

Reply:
I'd say that test names (as for unit test if I understand correctly) should have the name of the tested function AND the result. Maybe with more complexe function you'll need to test if the function does the correct thing, but also exceptions. For example:
test_parse_header_complete_pass
test_parse_header_missing_field_pass
test_parse_header_wrong_field_fail
In this case, the first one check if a full header makes a good call and successful return, the second checks if a missing field still returns ok and the third one check that if you give a wring input it fails successfully.

Reply:
It makes big function names, but when testing it will be clearer when something fails.